tests/test_formats.py

```diff
@@ -66,6 +66,15 @@ def test_datetimeformat(dt, formatted_datetime):
     assert datetimeformat(dt) == formatted_datetime
 
 
+@pytest.mark.parametrize("dt, formatted_datetime", (
+    (datetime(2012, 3, 24, 23, 59, 7, 6, tzinfo=pytz.utc), "Saturday 24 March 2012 at 11:59pm GMT"),
+    (datetime(2012, 3, 25, 23, 59, 7, 6, tzinfo=pytz.utc), "Monday 26 March 2012 at 12:59am BST"),
```

Comment:
I had to double take at this one, although now I have it's clear that this is fine. But could we maybe swing these dates to _obvious_ extremes? My first thought when I saw two dates so close together is that there was some other interface to the function that would let you toggle the GMT/BST thing. I think if the dates were in January/July it'd be more obvious that the date is controlling the timezone rather than anything else.

Reply:
It's using the same examples as the test below, so we can compare the behaviour of the different filters. Perhaps an explanatory comment would be useful after all.